src/new_enums.py

```diff
@@ -0,0 +1,120 @@
+from enum import Enum, IntEnum, auto
+
+
+class PlayerState(IntEnum):         # useless
+    IDLE = 0
+    WALK = 1
+
+
+class ItemToUse(IntEnum):         # useless
+    REGULAR_TOOL = 0
```

Comment:
This enum exists in case additional options have to be added, and fit much better than just using strings, since again, no typo risk, yadda, yadda